scripts/aether_functions.sh

```diff
@@ -32,18 +32,16 @@ function create_docker_assets {
     ./scripts/generate_env_vars.sh
 
     echo "${LINE} Generating docker network and database volume..."
-
-    docker network rm aether_bootstrap_net || true
     {
         docker network create aether_bootstrap_net \
             --attachable \
             --subnet=${NETWORK_SUBNET} \
             --gateway=${NETWORK_GATEWAY}
     } || true
-    echo "aether_bootstrap_net network is ready."
+    echo "${LINE} aether_bootstrap_net network is ready"
```

Comment:
`$LINE` is actually too long for my terminal... I wonder if there's a way to make it fit properly? I think `$COLUMNS` gives the width of the terminal, so maybe it would be possible to create a function that takes a string, and then left pads the string with something like `($COUMNS - length(string)) * '-'`?

Reply:
```bash
LINE=`printf -v row "%${COLUMNS:-$(tput cols)}s"; echo ${row// /=}`

function echo_message {
    if [ -z "${1:-}" ]; then
        echo "$LINE"
    else
        msg=" ${1:-} "
        echo "${LINE:${#msg}}$msg"
    fi
}

echo_message "whatever you want to say"
```
```text
========================================================== whatever you want to say
```